### PR TITLE
Feature/share groups

### PR DESCRIPTION
**Pull request by fibelatti**

### CODE REVIEW

**Comment by fibelatti** [comment]
This library is interesting if json validation is necessary: https://github.com/daveclayton/json-schema-validator (usage example: http://stackoverflow.com/questions/14511468/java-android-validate-string-json-against-string-schema)

**Comment by fibelatti** [comment]
@AndreFRSales @filipemp

File: app/src/main/java/com/fibelatti/raffler/views/activities/GroupActivity.java [hunk 1]
```diff
@@ -250,4 +258,22 @@ private void startSubGroupsActivity(Group group) {
...
+    private void shareGroup(Group group) {
+        FileHelper fileHelper = new FileHelper(context);
+        Intent shareIntent = new Intent();
+
+        if (fileHelper.createFileFromGroup(group)) {
+            Uri uri = FileProvider.getUriForFile(context,
+                    getString(R.string.file_provider_authority),
+                    new File(fileHelper.getGroupFilePath()));
+
+            shareIntent.setAction(Intent.ACTION_SEND);
```
Comment (AndreFRSales) [inline app/src/main/java/com/fibelatti/raffler/views/activities/GroupActivity.java]: @fibelatti could create one method for this intent. And sending the uri like a parameter

File: app/src/main/java/com/fibelatti/raffler/views/utils/FileHelper.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        Group group = new Group();
+
+        try {
+            JSONObject groupJson = new JSONObject(json);
+            JSONArray groupItemsJson = groupJson.getJSONArray(jsonPropertyGroupItems);
+
+            group.setName(groupJson.getString(jsonPropertyGroupName));
+
+            for (int i = 0; i < groupItemsJson.length(); i++) {
+                group.getItems().add(new GroupItem(groupItemsJson.get(i).toString()));
```
Comment (AndreFRSales) [inline app/src/main/java/com/fibelatti/raffler/views/utils/FileHelper.java]: @fibelatti Could create one method called : addGroupItem. And inside this method you could getItems().add(groupiTEM);

**Pull request merged**